Mathlib/Analysis/Normed/Ring/Basic.lean

```diff
@@ -642,9 +642,21 @@ variable {R₁ R₂ : Type*}
 for a continuous semilinear map to be bounded and this is the main use for this typeclass. -/
 class RingHomIsometric [Semiring R₁] [Semiring R₂] [Norm R₁] [Norm R₂] (σ : R₁ →+* R₂) : Prop where
```

Comment:
What do you think about renaming the class too to `RingHom.IsIsometric`?

Reply:
That sounds very reasonable for another PR, but I think we'd want to do the whole family of RingHom classes at the same time.